bodhi/client/__init__.py

```diff
@@ -453,7 +488,9 @@ def query(url, mine=False, rows=None, **kwargs):
         mine (Boolean): If the --mine flag was set
         kwargs (dict): Other keyword arguments passed to us by click.
     """
+    kwargs.pop('debug')
     client = bindings.BodhiClient(base_url=url, staging=kwargs['staging'])
+
```

Comment:
This is an unrelated style change, so let's leave it as it is.

Reply:
ok